packages/scheduler/lib/daemons/daemon.ts

```diff
@@ -40,7 +41,14 @@ export abstract class SchedulerDaemon {
             }
             this.status = 'running';
             while (!this.abortSignal.aborted) {
-                await this.run();
+                await tracer.trace(`scheduler.${this.name.toLowerCase()}.run`, async (span) => {
+                    try {
+                        await this.run();
+                    } catch (err) {
+                        span?.setTag('error', err);
+                        throw err;
```

Comment:
I never remember if it's necessary, I feel like it should be done by default by the wrapper

Reply:
no, you are right. this is done by `trace`